### ISSUE TITLE
Differences between this library's key values & browser UI event key values

### ISSUE THREAD

**Issue by HerbCaudill** [JohannesKlauss/react-hotkeys-hook#964]
Thanks for this library! I spun my wheels a bit figuring this out, so I thought it might be worth discussing: 

The key values this library uses are slightly different from the [browser API values](https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values): In this library they're all lowercase instead of PascalCase, and the "Arrow" keyword is omitted. For example:

```ts
useHotkeys(
  "up",
  (e, handler) => {
    const key = handler.keys?.join("")
    console.log(key) // up
    console.log(e.key) // ArrowUp
})
```

This seems worth documenting clearly, perhaps on the [Advanced usage](http://react-hotkeys-hook.vercel.app/docs/documentation/advanced-usage) page, which suggests using the `ts-key-enum` package for type safety.[1]  That page led me to believe I could do this: 

```ts
import { Key } from "ts-key-enum"

const { Enter, ArrowUp } = Key

useHotkeys(
  [Enter, ArrowUp], 
  (_, handler) => {
    switch (handler.keys?.join("")) {
      case Enter: {
        // 💣 This won't work because the value is `enter`
        break
      }

      case ArrowUp: {
        // 💣 This won't work because the value is `up`
        break
      }
      
      // etc. 
    }
  }
)
```

That won't work as noted in the comments. 

What I've ended up doing instead of using `ts-key-enum` is just making my own pseudo-enum of (a subset of) values that match what this library uses:

```ts
export const Key = {
  enter: "enter",
  tab: "tab",
  down: "down",
  left: "left",
  right: "right",
  up: "up",
  end: "end",
  home: "home",
  pagedown: "pagedown",
  pageup: "pageup",
  backspace: "backspace",
  escape: "escape",
  delete: "delete",
  insert: "insert",
} as const
export type Key = (typeof Key)[keyof typeof Key]
``` 

which allows me to do things like this: 

```ts
const { enter, up } = Key

useHotkeys(
  [enter, up], 
  (_, handler) => {
    switch (handler.keys?.join("")) {
      case enter: {
        // ✅
        break
      }

      case up: {
        // ✅
        break
      }
      
      // etc. 
    }
  }
)
```

Maybe this library could export an enum like that (except with all the possible values), instead of recommending `ts-key-enum`. I'd be happy to submit a PR if that sounds like a good idea. 

- - -
[1] As a side note, this bit of documentation probably belongs on the [Using TypeScript](https://react-hotkeys-hook.vercel.app/docs/documentation/typescript) page, rather than on its own on the "Advanced usage" page.

**Comment by Akrion** [comment]
I also had to deal with similar issue but related to how react testing library works and their userEvent section (here is their [KeyMap](https://github.com/testing-library/user-event/blob/main/src/keyboard/keyMap.ts)).

I agree that things if possible should be more consistent and standardized.

**Comment by JohannesKlauss** [comment]
Yeah, I am all for standardizing. Since this became quite a popular hook, it should probably support both ways.

**Comment by HerbCaudill** [comment]
To be clear, I'm not necessarily recommending standardization. 

I think this library already does a good job of _accepting_ a wide range of reasonable values: You can write `useHotkeys("up",...)` or `useHotkeys("Up",...)` or `useHotkeys("ArrowUp",...)` etc., and that's good. 

What I think would be useful is a clear way of knowing what values `handler.keys` will report within the callback. Currently it reports `up` instead of `ArrowUp` etc., and I don't have a problem with that other than that (a) it's not documented [here](https://react-hotkeys-hook.vercel.app/docs/api/use-hotkeys#handler) and (b) it doesn't match the values in the [officially recommended](https://react-hotkeys-hook.vercel.app/docs/documentation/advanced-usage) `ts-key-enum` package. 

In the future, it might make sense to "standardize" the `handler.keys` values, by using the same values that the browser event reports; but that would be a breaking change and would need to wait for a major release. (Or, I suppose it could be done behind an option flag.)

For now, I'd suggest addressing the two specific problems listed above by (a) adding documentation about `handler.keys` values, and (b) exposing a type representing the possible values returned by `handler.keys`. Let me know if a PR would be welcome.

**Comment by JohannesKlauss** [comment]
If you have time please go ahead and create a PR for this.
